java/ingest/ingest-trackerV2/src/test/java/sleeper/ingest/trackerv2/job/DynamoDBIngestJobTrackerCreatorIT.java

```diff
@@ -34,15 +34,18 @@ public class DynamoDBIngestJobTrackerCreatorIT extends LocalStackTestBase {
     @Test
     public void shouldCreateStore() {
         // When
-        DynamoDBIngestJobTrackerCreator.create(instanceProperties, dynamoClient);
+        DynamoDBIngestJobTrackerCreator.create(instanceProperties, dynamoClientV2);
 
         // Then
-        assertThat(dynamoClient.describeTable(tableName))
-                .extracting(DescribeTableResult::getTable).isNotNull();
+        assertThat(tableName.equals(
+                dynamoClientV2.describeTable(DescribeTableRequest.builder()
+                        .tableName(tableName)
+                        .build())
+                        .table().toString()));
```

Comment:
This isn't asserting anything. The assertThat method takes the actual value, and then you call a method on that to provide the expected value.